src/main/java/main/Main.java

```diff
@@ -0,0 +1,40 @@
+package main;
+
+import displays.Console;
+import httpApplication.SimpleApplication;
+import httpserver.*;
+import server.CommandLinePortValidator;
+import server.HostConnection;
+import server.Logger;
+import server.Server;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+
+        CommandLinePortValidator validator = new CommandLinePortValidator(5000);
+        final Integer SPECIFIED_PORT = validator.parsePort(args);
+
+        ServerSocket hostSocket = new ServerSocket(SPECIFIED_PORT);
+        HostConnection hostServer = new HostConnection(hostSocket);
+        Logger logger = new Logger(new Console(System.out));
+        ThreadPoolExecutor threadHandler =
+                new ThreadPoolExecutor(100, 100, 100, TimeUnit.MILLISECONDS, new ArrayBlockingQueue<Runnable>(100));
+
+        HttpServer httpProcess = new HttpServer(new SimpleApplication());
+
+        Server tcpServer = new Server(hostServer, threadHandler, httpProcess);
```

Comment:
Hey! Should `Server` be renamed to `TCPServer`?

Reply:
Possibly, what if it handled UDP connections too? Would that be more than one responsibility or would it fall under connecting a client to a hub?

Reply:
I am not sure. I am really just probing to see if there a more specific name. `Process` and `Server` are very generic. Generic names are good for interfaces/abstractions, but it is a little awkward when both the abstraction and the concretion are generic.

Reply:
I see, I'll change it here and see if there's other places I can add more specific names.